eras/alonzo/impl/testlib/Test/Cardano/Ledger/Alonzo/ImpTest.hs

```diff
@@ -12,23 +12,35 @@
 module Test.Cardano.Ledger.Alonzo.ImpTest (
   module ImpTest,
   initAlonzoImpNES,
-  PlutusArgs (..),
-  impAddPlutusScript,
-  impLookupPlutusScript,
-  fixupPPHash,
-  fixupRedeemers,
+  impLookupPlutusScriptMaybe,
   addCollateralInput,
-  alonzoFixupTx,
   impGetPlutusContexts,
+  alonzoFixupTx,
+  plutusTestScripts,
+  -- Fixup
   fixupDatums,
+  fixupOutputDatums,
+  fixupPPHash,
+  fixupRedeemers,
 ) where
 
 import Cardano.Crypto.DSIGN.Class (DSIGNAlgorithm (..))
 import Cardano.Crypto.Hash (Hash)
 import Cardano.Ledger.Address (Addr (..))
 import Cardano.Ledger.Alonzo (AlonzoEra)
-import Cardano.Ledger.Alonzo.Core (AlonzoEraScript (..), AlonzoEraTxOut (..), AsIxItem, Era (..), EraGov)
-import Cardano.Ledger.Alonzo.PParams (AlonzoEraPParams, getLanguageView, ppCostModelsL, ppMaxTxExUnitsL, ppMaxValSizeL)
+import Cardano.Ledger.Alonzo.Core (
+  AlonzoEraScript (..),
+  AlonzoEraTxOut (..),
+  AsIxItem,
+  Era (..),
+ )
```

Comment:
I suggest having an open Core import, this will avoid a whole bunch of redundant imports, thus reduce the amount if work we need to do when dealing with imports and resolving conflicts when working on the same module
```suggestion
import Cardano.Ledger.Alonzo.Core
```